cmd/ctl/pkg/version/version.go

```diff
@@ -79,16 +93,52 @@ func (o *Options) Validate() error {
 	}
 }
 
+// Complete takes the command arguments and factory and infers any remaining options.
+func (o *Options) Complete(factory cmdutil.Factory) error {
+	if o.ClientOnly {
+		return nil
+	}
+
+	restConfig, err := factory.ToRESTConfig()
+	if err != nil {
+		return fmt.Errorf("Error: cannot create the REST config: %v", err)
+	}
+
+	o.VersionChecker, err = versionchecker.New(restConfig, scheme.Scheme)
+	if err != nil {
+		return fmt.Errorf("Error: %v", err)
+	}
```

Comment:
These errors get passed to `cmdutl.CheckErr` which apparently adds its own `error: ` prefix to unrecongnized errors

```
// CheckErr prints a user friendly error to STDERR and exits with a non-zero
// exit code. Unrecognized errors will be printed with an "error: " prefix.
//
// This method is generic to the command in use and may be used by non-Kubectl
// commands.
func CheckErr(err error) {
	checkErr(err, fatalErrHandler)
}

```